ampeg/src/main/java/driverFactory/DriverManager.java

```diff
@@ -0,0 +1,38 @@
+package driverFactory;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+public abstract class DriverManager {
+	private WebDriver driver;
+	
+	public abstract void createDriver();
+	
+	public WebDriver getDriver() {
+		return driver;	
+	}
+	
+	protected void setDriver() {
+		driverSetup();
+		var url = "https://www.selenium.dev/";
```

Comment:
DriverManager should be generic enough to use when testing any web application. Remove url from this class. Suggest removing implicitwait setting also.